forage-android/src/main/java/com/joinforage/forage/android/ForageSDK.kt

```diff
@@ -12,24 +12,24 @@ import com.joinforage.forage.android.network.TokenizeCardService
 import com.joinforage.forage.android.network.data.CapturePaymentRepository
 import com.joinforage.forage.android.network.data.CheckBalanceRepository
 import com.joinforage.forage.android.network.model.ForageApiResponse
+import com.joinforage.forage.android.ui.ForagePANEditText
 import com.joinforage.forage.android.ui.ForagePINEditText
 import java.util.UUID
 
 /**
  * Singleton responsible for implementing the SDK API
  */
 object ForageSDK : ForageSDKApi {
-    private var panEntry: String = ""
     private val logger = Log.getInstance()
 
+    // TODO: this should be a Config argument that uses the builder pattern
     override suspend fun tokenizeEBTCard(
         merchantAccount: String,
+        panForageEditText: ForagePANEditText,
```

Comment:
thoughts on using the `forage` prefix (consistent with all of our SDKs and match var name closer to its type?

```suggestion
        foragePanEditText: ForagePANEditText,
```